models/Document/Editable/Scheduledblock.php

```diff
@@ -200,7 +200,7 @@ public function blockConstruct()
     /**
      * {@inheritdoc}
      */
-    public function blockStart($showControls = true, $return = false)
+    public function blockStart($showControls = true, $return = false, $additionalClass = '')
```

Comment:
```suggestion
    public function blockStart($showControls = true, $return = false)
```

Reply:
Not sure if this is possible, because `Scheduledblock extends Block` and while it's allowed to add additional parameters, I don't think it's allowed to remove any.

Reply:
If I remove this parameter I get an "Declaration must be compatible with ..." error.
So I just added it and don't use it just like `$return` and `$showControls`

Reply:
Yes, you are right. Thank you